skimage/feature/util.py

```diff
@@ -67,6 +68,10 @@ def plot_matches(ax, image1, image2, keypoints1, keypoints2, matches,
         color is chosen randomly.
     only_matches : bool, optional
         Whether to only plot matches and not plot the keypoint locations.
+    alignment : str, optional
+        Valid values are: 'horizontal' or 'vertical'
+        Whether to show images side by side, 'horizontal', or one above
+        the other, 'vertical'.
```

Comment:
If you want to highlight the parameter values, use \`\` instead of ' (e.g. \`\`horizontal\`\`).

Reply:
I might even say 

    ``'horizontal'``

That will show the string *with the single quotes* in monospace font.